deploy/init.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -eux
+
+# TODO Doesn't scale, find another way to wait until db is ready
+if [[ "${PANTSU_DBTYPE}" = "postgres" ]]; then
+  echo 'Waiting for the database to be ready...'
+  sleep 40
+fi
+
+go get github.com/ewhal/nyaa
```

Comment:
Guess we won't need it for long with the revert on templates, should I add it or wait for the revert PR to be merged ?